R/opq.R

```diff
@@ -18,6 +19,9 @@
 #' \dontrun{
 #' overpass_query(q) -> reading_noms
 #' sp::plot(reading_noms$osm_nodes)
+#' p <- overpass_query(opq("Leeds") %>%
+#'   add_feature("leisure", "park"))
+#' sp::plot(p$osm_polygons[1,])
```

Comment:
P.s. I think the result for "parks" should definitely be a SpatialPolygonsDataFrame - good use case of when autodetecting of most likely desired output is good - it should decide for you unless specified by an argument, e.g. `output_type = "points"` if the user really wants parks that OSM contributors have erroneously added as points.